docs/subnets/evm-customization/add-custom-precompiles/custom-precompiles.md

```diff
@@ -0,0 +1,67 @@
+# Custom Precompiles
+
+Avalanche allows you to add custom precompiles to your EVM. Adding precompiles instead of
```

Comment:
we should be more clear about precompiles vs stateful precompiles here. It's possible to add EVM precompiles (non-stateful ones) in Avalanche (Subnet-EVM more specifically), but that's not unique to us. This is supported by almost all EVM related VMs. 

We also do not have any out-of-the-box tutorial/tool to add new non-stateful precompiles. This is very advanced topic and could be hard for those reading these tutorial. For instance adding a new precompile would require coordinating new upgrade. This is easily covered by stateful precompiles by our module registration, but we don't cover non-stateful precompiles and it can be hard to add a new network upgrade. 

So we should probably focus on stateful precompiles rather than non-stateful ones. We can still implement the very same non-stateful precompiles (like sha256 etc) with stateful precompiles.